external-crates/move/crates/move-compiler/tests/custom_rules/constant_naming.move

```diff
@@ -0,0 +1,20 @@
+module 0x42::M {
+    // Correctly named constants
+    const MAX_LIMIT: u64 = 1000;
+    const MIN_THRESHOLD: u64 = 10;
+    const MIN_U64: u64 = 10;
```

Comment:
Could we split this into two separate test files. One with all the valid cases and another with all of the invalid ones?

Reply:
yes.We could. refactored